.github/workflows/test-deploy-wf-service.yml

```
@@ -25,17 +25,185 @@ on:
         description: 'Environment'
         required: true
         default: 'dev'
-      tag:
+      sha:
         type: string
-        description: 'Image Tag'
+        description: 'Commit SHA ID'
         required: true
         default: ''
```

actionlint

input "sha" of workflow_call event has the default value "", but it is also required. if an input is marked as required, its default value will never be used

(events)